db/migrate/20180409120422_create_physical_network_ports.rb

```diff
@@ -0,0 +1,19 @@
+class CreatePhysicalNetworkPorts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :physical_network_ports do |t|
+      t.bigint  :ems_ref
+      t.string  :uid_ems
+      t.string  :port_name
+      t.string  :port_type
+      t.string  :peer_mac_address
+      t.string  :vlan_key
+      t.string  :mac_address
+      t.integer :port_index
+      t.boolean :vlan_enabled
+      t.bigint  :guest_device_id
+      t.bigint  :switch_id
```

Comment:
is this relating to the switches table or the physical_switches table?  If the latter, this should be physical_switch_id.

Reply:
Oh nevermind...For some reason I thought there was a physical_switches table separate from switches.